### DISCUSSION TITLE
Flag to disable creation of paper.yml-README.txt file.

### DISCUSSION THREAD

**Discussion by RealTriassic** [PurpurMC/Purpur#1113] [__Dismissed__] [answered]
I think it could be nice if there was an optional Java flag available that prevents the `paper.yml-README.txt` file from being created in the server's root directory, helps keep the server files clean.

**Comment by hexadecimal233** [comment]
You should ask paper team first

**Comment by granny** [comment]
Have you already asked Paper to implement it? You can do that by either creating an issue on their github page or by creating a pull request that implements that feature.

I think a better solution would be to only generate the README when the config files are being generated, so that it can be deleted by the user once the config files are in place.

**Comment by RealTriassic** [comment, marked as answer]
Thanks for the responses, I have went ahead and opened a feature request on Paper's repository over [here](https://github.com/PaperMC/Paper/issues/8332).
